aggregator/src/test/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJobTest.java

```diff
@@ -4,17 +4,15 @@
 import com.redhat.cloud.notifications.db.EmailAggregationResources;
 import com.redhat.cloud.notifications.db.ResourceHelpers;
 import com.redhat.cloud.notifications.models.AggregationCommand;
-import com.redhat.cloud.notifications.models.EmailSubscriptionType;
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusTest;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import javax.inject.Inject;
-import java.time.Duration;
-import java.time.Instant;
 import java.util.List;
 
```

Comment:
This class needs to test the messages emitted through SmallRye Reactive Messaging using `InMemoryConnector`.

Reply:
I changed every of your findings besides the missing InMemoryConnector test. I will add it tomorrow morning.

Reply:
There are a few `hidden conversations` above with comments/questions. 😃

Reply:
I didn't hide these. Thought someone of you did it and don't want to talk about it anymore.